### ISSUE TITLE
Use importlib.metadata if it exists

### ISSUE THREAD

**Issue by jayvdb** [sinoroc/tox-poetry-dev-dependencies#17]
Python 3.8 does not need `importlib_metadata` - it is part of stdlib as `importlib.metadata`.

```py
  File "/usr/lib/python3.8/site-packages/tox_poetry_dev_dependencies/__init__.py", line 5, in <module>
    from . import _meta
  File "/usr/lib/python3.8/site-packages/tox_poetry_dev_dependencies/_meta.py", line 5, in <module>
    import importlib_metadata
ModuleNotFoundError: No module named 'importlib_metadata'
```

**Comment by sinoroc** [comment]
Why the stack trace? How did this happen? There is a mandatory dependency on `importlib-metadata`, I do not see how this could have happened.

**Comment by sinoroc** [comment]
@jayvdb 
Since `importlib_metadata` is used exclusively to provide the value for `__version__`, accodingly to [PEP 396](https://www.python.org/dev/peps/pep-0396/), and my impression is that no one actually relies on the presence of this value (additionally the PEP itself never went to the final stage), it might be worth it getting rid of `__version__` entirely.

Is that something that would work for you?

Do you rely on PEP 396 (the presence of `__version__`), or are you aware of any project that does?

The maintainers of _importlib-metadata_ themselves got rid of it in this [change](https://gitlab.com/python-devs/importlib_metadata/-/merge_requests/125).

**Comment by jayvdb** [comment]
I use a different tox poetry plugin now.

**Comment by sinoroc** [comment]
> I use a different tox poetry plugin now.

@jayvdb For info, which plugin are you using? I want to link to it in the README, so that people can find alternatives easily.

These are the ones I have so far:

* https://pypi.org/project/tox-poetry-installer/
* https://pypi.org/project/tox-poetry/

**Comment by sinoroc** [comment]
For info, the removal of `__version__` and thus the dependency on _importlib-metadata_ has been released in [v0.0.4](https://pypi.org/project/tox-poetry-dev-dependencies/0.0.4/).
